installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json

```diff
@@ -65,7 +65,8 @@
         "DEPLOYMENT_TARGETS": {
           "API": ["aws_apigateway", "gcp_apigee"],
           "FUNCTION": ["aws_lambda"],
-          "WEBSITE": ["aws_s3", "aws_cloudfront"]
+          "WEBSITE": ["aws_s3", "aws_cloudfront"],
+          "SLS-APP": ["aws_sls-app"]
```

Comment:
@olegfomindev9  I was thinking about this.. For API/function/website, we specifically know where Jazz deploys them to. For sls_app, it can be a combination of AWS services. Proposing to use aws_* here. Thoughts?

Reply:
We'll proceed this for now. We'll use more generic key-values when we have multiple frameworks to deal with.